vcpkg_overlays/h3/vcpkg.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "h3",
+  "version-semver": "3.7.2",
+  "port-version": 1,
+  "description": "A Hexagonal Hierarchical Geospatial Indexing System",
+  "homepage": "https://github.com/uber/h3",
+  "supports": "!uwp",
+  "dependencies": [
+    {
+      "name": "vcpkg-cmake",
+      "host": true
+    },
+    {
+      "name": "vcpkg-cmake-config",
+      "host": true
+    }
+  ]
+}
```

Comment:
Note:  This file will be removed prior to 9.4.0 (See:  https://github.com/hpcc-systems/HPCC-Platform/pull/17374)